source/calculus/source/01-LT/02.ptx

```diff
@@ -466,15 +517,45 @@ The limit is <m>-0.7</m>.
         </table>
 </introduction>
      <task> <statement> <p>What was the average velocity on the first 50 meters? On the second 50 meters? </p>
-         </statement> </task>  
+         </statement> 
+         <answer>
+          <p>
+            The average velocity on the first 50 meters is <m>9.09</m> meters per second  and   <m>11.93 </m> meters per second on the second 50 meters.
+          </p>
+         </answer>
+         </task>  
                  <task> <statement> <p>What was the average velocity between 30 and 50 meters? Between 50 and 70 meters? </p>
-                  </statement> </task>    
+                  </statement>
+                  <answer>
+          <p>
+            The average velocity between 30 and 50 meters is   <m>11.62 </m> meters per second and   <m>12.19 </m> meters per second between 50 and 70 meters.
+          </p>
+         </answer>
+                   </task>    
                  <task> <statement> <p> What was the average velocity between 40 and 50 meters? Between 50 and 60 meters? </p>
-         </statement> </task> 
+         </statement>
+          <answer>
+          <p>
+            The average velocity between 40 and 50 meters is   <m>11.76 </m> meters per second and   <m>12.20 </m> meters per second between 50 and 60 meters.
+          </p>
+         </answer>
+          </task> 
             <task> <statement> <p>What is your best estimate for the Usain's velocity at the instant when he passed the 50 meters mark? This is your estimate for the instantaneous velocity. </p>
-         </statement> </task> 
+         </statement> 
+          <answer>
+          <p>
+            <m>12</m> meter per second.
+          </p>
+         </answer>
+          </task> 
                <task> <statement> <p>Using the table of values, explain why 50 meters is NOT the best guess for when the instantaneous velocity was the largest. What other point would be more reasonable? </p>
-         </statement> </task> 
+         </statement>
+        <answer>
+          <p>
+            <m>12</m> 
```

Comment:
I think the intention is to have students analyze the average velocities on all 10 subintervals to see that he's still speeding up at 50 meters. Here's the data from excel: 
<img width="558" alt="table" src="https://github.com/user-attachments/assets/97e4289e-451a-4132-88f8-f32057100b19" />